packages/components/src/mobile/bottom-sheet/range-text-input.native.js

```diff
@@ -188,7 +188,10 @@ class RangeTextInput extends Component {
 		];
 
 		const valueFinalStyle = [
-			! isIOS ? inputBorderStyles : verticalBorderStyle,
+			Platform.select( {
+				android: inputBorderStyles,
+				ios: { ...verticalBorderStyle, color: textInputStyle.color },
```

Comment:
Seems like `verticalBorderStyle` is only used in `TextInput` and only for `iOS`? Maybe we can add `color: $gray-dark/$white` to `.verticalBorder` style instead and maybe we can even rename it to `.textInputIOS`?

Reply:
Fair point, I've updated this in 1ad0d9636c3177c8489f49ef7dfd140602f4df2a.